pkgs/desktops/cinnamon/default.nix

```diff
@@ -1,5 +1,16 @@
 { pkgs, lib }:
 
+
+let
+
+  # https://github.com/linuxmint/cjs/issues/80
+  spidermonkey_52 = callPackage ./spidermonkey_52.nix {};
+
+in
```

Comment:
I'd do this in the actual package expression.

Reply:
Unless, there's others that need this in cinnamon?

Reply:
How do I get callPackage working there?

Reply:
this 
```diff
diff --git a/pkgs/desktops/cinnamon/default.nix b/pkgs/desktops/cinnamon/default.nix
index 9da9db456a2..371015d02bb 100644
--- a/pkgs/desktops/cinnamon/default.nix
+++ b/pkgs/desktops/cinnamon/default.nix
@@ -4,7 +4,7 @@
 let
 
   # https://github.com/linuxmint/cjs/issues/80
-  spidermonkey_52 = callPackage ./spidermonkey_52.nix {};
+  spidermonkey_52 = pkgs.callPackage ./cjs/spidermonkey_52.nix {};
 
 in

 ```

But I believe you meant for this just to be in the `cjs` expression.